bnb/getBNBData.sh

```diff
@@ -1,11 +1,13 @@
 #!/bin/bash
 
-cd /var/www/Tools/BNBDaten
+BNB_DATADIR="/var/www/Tools/BNBDaten"
```

Comment:
Actually, this path might have changed on production or test environment. By saving the path in the variable will make it easier to change it as well. Thank you.